.github/workflows/build-test.yaml

```diff
@@ -0,0 +1,50 @@
+name: Build Test
+
+on:
+  push:
+    branches: [main]
+    paths-ignore:
+      - README.md
+  pull_request:
+    branches: [main]
+    paths-ignore:
+      - README.md
+
+jobs:
+  Build-Test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
```

Comment:
This removes the submodules right?

Reply:
yep, I just copied from another repo and forgot to change that